web/server.js

```diff
@@ -0,0 +1,261 @@
+var express = require('express');
+var app = express();
+var multer = require('multer')
+var cors = require('cors');
+var fs=require('fs')
+app.use(cors())
+var bodyParser = require('body-parser');
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(express.json())
+
+//For Patient 1
+var storage = multer.diskStorage({
+    destination: function (req, file, cb) {
```

Comment:
Instead of defining 4 endpoints in the server, one for each patient, you could define a single one that receives as argument the current patient and does all the computation generically.
As a result, you will not have duplicated code here, there will be fewer lines of code and this file would be easier to read. Thus, it will also be easier to maintain: if you need to do a change, you will not have to do it 4 times, in each of the endpoints, but a single time in the generic endpoint.